src/HttpRequest.php

```diff
@@ -3,6 +3,28 @@
 namespace Onoi\HttpRequest;
 
 /**
+ * @since 1.1
+ *
+ * @{
+ */
+// @codeCoverageIgnoreStart
+define( 'ONOI_HTTP_REQUEST_URL', 10 );
+define( 'ONOI_HTTP_REQUEST_PROTOCOL_VERSION', 11 );
+define( 'ONOI_HTTP_REQUEST_METHOD', 12 ); // POST, GET, HEAD
+
+define( 'ONOI_HTTP_REQUEST_CONNECTION_TIMEOUT', 21 );
+define( 'ONOI_HTTP_REQUEST_CONNECTION_FAILURE_REPEAT', 22 );
+
+define( 'ONOI_HTTP_REQUEST_CONTENT_TYPE', 31 ); // "Content-Type: ..."
+define( 'ONOI_HTTP_REQUEST_CONTENT', 32 );
+
+define( 'ONOI_HTTP_REQUEST_SSL_VERIFYPEER', 40 ); // SSL setting true, false
+define( 'ONOI_HTTP_REQUEST_STREAM_CLIENT_FLAGS', 50 ); // STREAM_CLIENT_ASYNC_CONNECT | STREAM_CLIENT_PERSISTENT
```

Comment:
>  rather than the 50 here?

It could have been any number, it is only to ensure that internally when accessing the options array via `$this->getOption( ONOI_HTTP_REQUEST_STREAM_CLIENT_FLAGS )` an accessors for example `$this->setOption( ONOI_HTTP_REQUEST_STREAM_CLIENT_FLAGS, STREAM_CLIENT_ASYNC_CONNECT | STREAM_CLIENT_CONNECT );` uses the same flag.

I could define `define( 'ONOI_HTTP_REQUEST_STREAM_CLIENT_FLAGS', 'ONOI_HTTP_REQUEST_STREAM_CLIENT_FLAGS' )` but internally relevant to the options array that keeps the settings.